src/components/CardList/index.jsx

```diff
@@ -0,0 +1,17 @@
+import Card from "components/Card";
+import styles from "./styles.module.css";
+
+export default function CardList({ links }) {
+  return (
+    // <div className={style.wrapper}>
+    <ul className={styles["l_row"]}>
+      {/* 옵셔널 체이닝: 이걸 이용하지 않으면 새로고침시 에러가 발생한다 */}
+      {links?.map((link) => (
+        <li key={link.id} className={styles["l_col"]}>
+          <Card link={link} />
+        </li>
+      ))}
+    </ul>
+    // </div>
+  );
```

Comment:
P2:
해당 컴포넌트를 가져다쓰는 folder에서는 link.length 를 확인해 값이 없을 시 이에 해당하는 컴포넌트를 보여주지만
share 페이지의 경우 위의 로직이 없어서 optional chaning이 필요하네요.
기왕이면 방어로직을 해당 컴포넌트 안에 넣고 통일성을 주면 좋겠습니다.


```suggestion

  return !links.length ?
    <NoData/> : (
    <ul className={styles["l_row"]}>
      {links.map((link) => (
        <li key={link.id} className={styles["l_col"]}>
          <Card link={link} />
        </li>
      ))}
    </ul>
  );
```